site/docs/4.4/components/forms.md

```diff
@@ -978,7 +978,7 @@ Validation styles are available for the following form controls and components:
 
 {% capture callout %}
 ##### Input group validations
-Input groups have difficulty with validation styles unfortunately. Our recommendation is to place feedback messages as sibling elements of the `.input-group` that has `.is-{valid|invalid}`. Placing feedback messages within input groups breaks the `border-radius`. [See this comment for workaround.](https://github.com/twbs/bootstrap/issues/25110#issuecomment-586565165)
+Input groups have difficulty with validation styles, unfortunately. Our recommendation is to place feedback messages as sibling elements of the `.input-group` that has `.is-{valid|invalid}`. Placing feedback messages within input groups breaks the `border-radius`. [See this comment for a workaround](https://github.com/twbs/bootstrap/issues/25110#issuecomment-586565165).
```

Comment:
@MartijnCuppens @mdo: I'm still not happy with us linking to a GitHub comment. We should move the workaround here.